src/main/java/edu/itu/cavabunga/core/factory/ComponentFactory.java

```diff
@@ -2,11 +2,75 @@
 
 import edu.itu.cavabunga.core.entity.Component;
 import edu.itu.cavabunga.core.entity.component.ComponentType;
+import edu.itu.cavabunga.core.exception.IcalStorageException;
+
+import java.util.Date;
 
 @org.springframework.stereotype.Component
```

Comment:
is it really necessary?

Reply:
How else we store date data format ? string ?

Reply:
I was talking about @org.springframework.stereotype.Component . Whatever:) it is necessary